src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -382,6 +382,8 @@ class PolicyHandler : public PolicyHandlerInterface,
                        const std::string& wers_country_code,
                        const std::string& language) OVERRIDE;
 
+  const std::string& GetCCPUVersion() OVERRIDE;
```

Comment:
@ydementieiev Why do you have the functions with the same meaning but different names? \
```cpp
const std::string& ccpu_version() const OVERRIDE;
const std::string& GetCCPUVersion() OVERRIDE;
```
The should have the same name or do something different in case of different names

Reply:
@AByzhynar  [renamed](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/0d60c4e63814fd53aa0e6c9e0f26893e9587efca#diff-17199700554f77ec5d6b4968cc85e7b9R385)